newton/_src/sim/builder.py

```diff
@@ -194,6 +194,10 @@ class ShapeConfig:
         If provided (and sdf_target_voxel_size is None), enables SDF-based mesh-mesh collision.
         Set to None (default) to disable SDF generation for this shape (uses BVH-based collision for mesh-mesh instead).
         Requires GPU since wp.Volume only supports CUDA. Only used for mesh shapes."""
+        is_hydroelastic: bool = False
+        """Whether the shape collides using SDF-based hydroelastics. For hydroelastic collisions, both participating shapes must have is_hydroelastic set to True. Defaults to False."""
```

Comment:
Maybe add a `.. note::` here explaining that hydroelastic collision handling only works with volumetric shapes, so this flag will be set to false for planes and heightfields in `ModelBuilder.add_shape()`.